src/schemas/execute.yaml

```diff
@@ -0,0 +1,284 @@
+    stateOverrides:
+      title: State overrides
+      $ref: '#/components/schemas/StateOverrides'
+    calls:
+      type: array
+      title: List of transactions to execute at this block/state
+      items:
+        $ref: '#/components/schemas/GenericCallTransaction'
+StateOverrides:
+  title: Accounts in state to be overridden
+  type: array
+  items:
+    $ref: '#/components/schemas/AccountOverride'
+AccountOverride:
+  title: Details of an account to be overridden
+  type: object
+  oneOf:
+    - $ref: '#/components/schemas/AccountOverrideState'
+    - $ref: '#/components/schemas/AccountOverrideStateDiff'
+AccountOverrideState:
+  title: Account override with whole storage replacement
+  required:
+    - state
+  properties:
+    address:
+      title: Address
+      $ref: '#/components/schemas/address'
+    nonce:
+      title: Nonce
+      $ref: '#/components/schemas/uint64'
+    balance:
+      title: Balance
+      $ref: '#/components/schemas/uint256'
+    code:
+      title: Code
+      $ref: '#/components/schemas/bytes'
+    moveToAddress:
+      title: MoveToAddress
+      $ref: '#/components/schemas/address'
+    state:
+      title: Storage
+      $ref: '#/components/schemas/AccountStorage'
+AccountOverrideStateDiff:
+  title: Account override with partial storage modification
+  required:
+    - stateDiff
+  properties:
+    address:
+      title: Address
+      $ref: '#/components/schemas/address'
+    nonce:
+      title: Nonce
+      $ref: '#/components/schemas/uint64'
+    balance:
+      title: Balance
+      $ref: '#/components/schemas/uint256'
+    code:
+      title: Code
+      $ref: '#/components/schemas/bytes'
+    moveToAddress:
+      title: MoveToAddress
+      $ref: '#/components/schemas/address'
+    stateDiff:
+      title: Storage difference
+      $ref: '#/components/schemas/AccountStorage'
+AccountStorage:
+  title: Storage slots for an account
+  type: object
+  additionalProperties:
+    - $ref: '#/components/schemas/hash32'
+BlockOverrides:
+  title: Context fields related to the block being executed
+  type: object
+  properties:
+    number:
+      title: Number
+      $ref: '#/components/schemas/uint256'
+    prevRandao:
+      title: Randomness beacon
+      $ref: '#/components/schemas/uint256'
+    time:
+      title: Time
+      $ref: '#/components/schemas/uint256'
+    gasLimit:
+      title: Gas limit
+      $ref: '#/components/schemas/uint64'
+    feeRecipient:
+      title: feeRecipient
+      $ref: '#/components/schemas/address'
+    baseFeePerGas:
+      title: Base fee per gas
+      $ref: '#/components/schemas/uint256'
+MultiCallResult:
+  title: Full results of multi call
+  type: array
+  items:
+    $ref: '#/components/schemas/MultiCallBlockResult'
+MultiCallBlockResult:
+  title: Result of multicall block-level, with array of calls
+  type: object
+  properties:
+    number:
+      title: Number
+      $ref: '#/components/schemas/uint256'
+    hash:
```

Comment:
If possible I'd like us to remove this field. I feel it'll be a random hash that doesn't have a use-case anyway and to produce it we have to merkleize the state as well as receipts and withdrawals.

Reply:
I believe blocks still need a hash, because the `BLOCKHASH(number)` opcode has to return something.  I think there is value in letting the caller know what value would have been returned by that opcode is useful.